packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -4,167 +4,508 @@
-    required this.surface,
-    required this.background,
-    required this.error,
     required this.onPrimary,
+    Color? primaryContainer,
+    Color? onPrimaryContainer,
+    required this.secondary,
     required this.onSecondary,
-    required this.onSurface,
-    required this.onBackground,
+    Color? secondaryContainer,
+    Color? onSecondaryContainer,
+    Color? tertiary,
+    Color? onTertiary,
+    Color? tertiaryContainer,
+    Color? onTertiaryContainer,
+    required this.error,
     required this.onError,
-    required this.brightness,
-  }) : assert(primary != null),
-       assert(primaryVariant != null),
-       assert(secondary != null),
-       assert(secondaryVariant != null),
-       assert(surface != null),
-       assert(background != null),
-       assert(error != null),
+    Color? errorContainer,
+    Color? onErrorContainer,
+    required this.background,
+    required this.onBackground,
+    required this.surface,
+    required this.onSurface,
+    Color? surfaceVariant,
+    Color? onSurfaceVariant,
+    Color? outline,
+    Color? shadow,
+    Color? inverseSurface,
+    Color? onInverseSurface,
+    Color? inversePrimary,
+    @Deprecated(
+      'Use primary or primaryContainer instead. '
+      'This feature was deprecated after v2.6.0-0.0.pre.'
+    )
+    Color? primaryVariant,
+    @Deprecated(
+      'Use secondary or secondaryContainer instead. '
+      'This feature was deprecated after v2.6.0-0.0.pre.'
+    )
+    Color? secondaryVariant,
+  }) : assert(brightness != null),
+       assert(primary != null),
        assert(onPrimary != null),
+       assert(secondary != null),
        assert(onSecondary != null),
-       assert(onSurface != null),
-       assert(onBackground != null),
+       assert(error != null),
        assert(onError != null),
-       assert(brightness != null);
+       assert(background != null),
+       assert(onBackground != null),
+       assert(surface != null),
+       assert(onSurface != null),
+       _primaryContainer = primaryContainer,
+       _onPrimaryContainer = onPrimaryContainer,
+       _secondaryContainer = secondaryContainer,
+       _onSecondaryContainer = onSecondaryContainer,
+       _tertiary = tertiary,
+       _onTertiary = onTertiary,
+       _tertiaryContainer = tertiaryContainer,
+       _onTertiaryContainer = onTertiaryContainer,
+       _errorContainer = errorContainer,
+       _onErrorContainer = onErrorContainer,
+       _surfaceVariant = surfaceVariant,
+       _onSurfaceVariant = onSurfaceVariant,
+       _outline = outline,
+       _shadow = shadow,
+       _inverseSurface = inverseSurface,
+       _onInverseSurface = onInverseSurface,
+       _inversePrimary = inversePrimary,
+       _primaryVariant = primaryVariant,
+       _secondaryVariant = secondaryVariant;
+
+  /// Generate a [ColorScheme] based around the given `seedColor`.
+  ///
+  /// Using the seedColor as a starting point, a set of tonal palettes are
```

Comment:
Is there a way to generate a high-contrast color scheme from a seed color? It would help to provide a path from _all_ of the existing factories, to ColorScheme.fromSeed().

Reply:
I didn't see an obvious way to do this with `material_color_utilities`, but @guidezpl would know.

Reply:
Sorry for the late reply.. There isn't